oap-server/server-starter/src/main/resources/component-libraries.yml

```diff
@@ -435,6 +435,10 @@ Impala:
   id: 134
   languages: Java
 
+EventMesh-producer:
```

Comment:
Could you explain why only producer? Where is consumer and server?
And why do you need producer rather than client?

Reply:
As you are adding a tracing plugin, you have to do both sides, otherwise, the tracing context breaks, and ppl would feel confused.